osmcrops.csv

```diff
@@ -1,77 +1,77 @@
-Chestnut,trees=chestnut_trees,orchard,also maybe crop=nut?
-Hazelnuts- with shell,trees=hazel_plants,orchard,also maybe crop=nut?
-Pistachios,trees=pistachio_trees,orchard,also maybe crop=nut?
-Peaches and nectarines,trees=peach_trees,orchard,"Wiki-EN: “Peach and nectarines are the same species, even though they are regarded commercially as different fruits.”"
-Persimmons,trees=persimmon_trees,orchard,
-Plums and sloes,trees=plum_trees,orchard,"A hybrid, basically the same, does not matter here anyway"
-Rubber- natural,trees=rubber_trees,orchard,
-Oil- palm,trees=oil_palms,orchard,
-Oil- palm fruit,trees=oil_palms,orchard,
-Oilseeds,trees=oil_palms,orchard,
-Palm kernels,trees=oil_palms,orchard,usually the “palm cake” is only a side product of palm oil production; tagging them differently or as “palms” (unspecific) could be confusing
-Coconuts,trees=coconut_palms,orchard,also maybe crop=nut?
-Dates,trees=date_palms,orchard,
-Blueberries,trees=blueberry,orchard,
-???????,trees=bayberry,orchard,
-???????,trees=blackberry,orchard,
-Raspberries,trees=raspberry,orchard,
-Strawberries,crop=strawberry,orchard,
-???????,trees=coca_plants,orchard,
-Cocoa- beans,trees=cacao_trees,orchard,"Note different spelling (cf.: trees=cacao_trees); both spellings possible, but tag used here is more common in OSM"
-Pepper (piper spp.),trees=piper_plants,orchard,
-???????,trees=nopal_plants,orchard,
-???????,trees=tamarind,orchard,wrong syntax?
-Pineapples,trees=pineapple_plants,orchard,incorrect: trees=ananas_plants
-???????,trees=shal_trees,orchard,https://en.wikipedia.org/wiki/Shorea_robusta
-???????,trees=blackcurrant_plants,orchard,
-Tangerines- mandarins- clementines- satsumas,trees=orange_trees,orchard,could also be mapped each one under: trees=mandarin_trees
-???????,trees=lychee_trees,orchard,
-???????,trees=sand_pear,orchard,
-???????,trees=eucalyptus_trees,orchard,
-???????,trees=pitaya_plants,orchard,
-???????,trees=cherimoya_trees,orchard,
-???????,trees=pomegranate_trees,orchard,
-???????,trees=pecan_trees,orchard,also maybe crop=nut?
-# CURRENTLY MISSING/UNOFFICIAL/PROPOSALS BELOW,,orchard,
-Chillies and peppers- green,crop=pepper,orchard,
-Chillies and peppers- dry,crop=pepper,orchard,
-Nutmeg- mace and cardamoms,trees=nutmeg_trees,orchard,
-Vanilla,crop=vanilla,orchard,
-Sisal,crop=sisal,farmland,
-Cinnamon,crop=cinnamon,orchard,"grows ‘on’ trees (bark), maybe not really an orchard"
-Kola nuts,trees=kola_trees,orchard,
-Maté,trees=mate_plants,orchard,
-Jojoba seed,trees=jojoba_plants,orchard,
-Cranberries,trees=cranberry,orchard,
-Grapefruit (inc. pomelos),trees=grapefruit_trees,orchard,
-Tomatoes,crop=tomatoe,MAYBE building=greenhouse,
+crop name,key (OSM produce value),OSM trees-key value,landuse key,comment
+Apples,apple,apple_trees,orchard,
+Bananas,banana,banana_plants,orchard,
+Coffee,coffee,coffee_plants,orchard,
+Coffee- green,coffee,coffee_plants,orchard,
+Grapes,grape,,vineyard,should retag as landuse=vineyard

```

Comment:
Hmm, a vineyard could also "produce=wine"… :wink:
Not that easy this "produce" tag too.